topostats/grains.py

```diff
@@ -148,20 +198,23 @@ def calc_minimum_grain_size(self, image: np.ndarray) -> float:
         else:
             self.minimum_grain_size = -1
 
-    def remove_noise(self, image: np.ndarray, **kwargs) -> np.ndarray:
-        """Remove noise which are objects smaller than the 'smallest_grain_size_nm2'.
+    def remove_noise(self, image: npt.NDArray, **kwargs) -> npt.NDArray:
+        """
+        Remove noise which are objects smaller than the 'smallest_grain_size_nm2'.
```

Comment:
Would it be better to write something like "Removes noise, where noise is defined as objects smaller than the 'smallest_grain_size_nm2"?

Reply:
I was pretty lazy on this as the `smallest_grain_size_nm2` is due to be removed and the `remove_noise()` function isn't actually called anywhere at present (I started work on removing this the other day too see #805 )